react/typings/globals.d.ts

```diff
@@ -0,0 +1,117 @@
+import { ComponentType } from 'react'
+import { MaybeResponsiveInput } from 'vtex.responsive-values'
+
+declare global {
+  interface GalleryLegacyProps {
```

Comment:
If we import the React components from another app the global types are not imported (like we will miss the feature that we are using in vtex.css-handles and other apps)

So we should not use `globals.d.ts`

Reply:
Got it. Should we set these types in the components they belong to, then? Feels kinda bad to mix regular imports with type imports 😔

Reply:
You can use `import type { FooBar } from './module'`